### PR TITLE
added import from cunit basic.h

### PR DESCRIPTION
**Pull request by CatherineZeng**
added #include <CUnit/Basic.h>

### CODE REVIEW

File: tests/testCCRing.c [hunk 1]
```diff
@@ -15,6 +16,14 @@ void should_fail(void) {
...
 }
 
+int test array(array1, array2, ) {
+  if =
+    return 1
+  if != 
+    return 0;
+}
+
+
```
Comment (CharlesHolbrow) [inline tests/testCCRing.c]: I don't think you meant to include  this pseudo code. Make sure to `$ git diff` every time before you `$ git add` files so you know exactly what you are changing.

File: tests/testCCRing.c [hunk 2]
```diff
@@ -3,6 +3,7 @@
 // tests
 #include <CUnit/CUnit.h>
 #include <CUnit/Console.h>
+#include <CUnit/Basic.h>
```
Comment (CharlesHolbrow) [inline tests/testCCRing.c]: Great job

File: tests/testCCRing.c [hunk 3]
```diff
@@ -39,5 +48,7 @@ int main(int argc, char** argv) {
   CU_basic_run_tests(); // results to stdout
   CU_cleanup_registry();
 
+  printf("testing");
```
Comment (CharlesHolbrow) [inline tests/testCCRing.c]: same as above!

**Comment by CatherineZeng** [comment]
Oh dear I totally missed this!

Thanks :) I will be more careful and check next time.

Enviado desde mi iPhone

**Comment by CharlesHolbrow** [comment]
No worries! When you have time, if you fix/push a working version to your github fork, I think that github will give you an option to update your pull request.

**Pull request merged**